demisto_sdk/commands/validate/tests/RM_validators_test.py

```diff
@@ -53,3 +56,44 @@ def test_IsContainCopyRightSectionValidator_is_valid(
             for result, expected_msg in zip(results, expected_msgs)
         ]
     )
+
+
+@pytest.mark.parametrize(
+    "content_items, expected_number_of_failures, expected_msg",
+    [
+        (
+            [
+                create_metadata_object(
+                    readme_text="This readme text and pack_metadata description are equal",
+                    paths=["description"],
+                    values=["This readme text and pack_metadata description are equal"],
+                ),
+                create_metadata_object(
+                    readme_text="Readme text",
+                    paths=["description"],
+                    values=["Pack_metadata description"],
+                ),
+            ],
+            1,
+            "README.md content is equal to pack description. Please remove the duplicate description from README.md file.",
+        )
+    ],
+)
+def test_IsPackReadmeNotEqualPackDescriptionValidator(
```

Comment:
please hardcode the values into the test instead of using `parametrize` for a single test case